src/components/Advertisements/Advertisements.tsx

```diff
@@ -25,65 +19,7 @@ export default function advertisements(): JSX.Element {
   const { t } = useTranslation('translation', { keyPrefix: 'advertisement' });
   document.title = t('title');
 
```

Comment:
Instead of assigning numbers to the loading variables, you could simply use names that correspond to the data being loaded. For example, instead of **loading2** you could name it **loadingAdvertisements**